src/main/java/org/apache/mesos/scheduler/repair/CoreFrameworkState.java

```diff
@@ -0,0 +1,44 @@
+package org.apache.mesos.scheduler.repair;
+
+import org.apache.mesos.Protos.FrameworkID;
+import org.apache.mesos.Protos.TaskID;
+import org.apache.mesos.Protos.TaskInfo;
+import org.apache.mesos.state.StateStoreException;
+
+import java.util.List;
+
+/**
+ * Exposes simple task-oriented API for components that need to interact with the framework
+ * as a whole.
+ */
+public interface CoreFrameworkState {
```

Comment:
I created this interface between there's no interface for accessing and modifying live tasks, only the StateStore. Would you like me to split this into one new interface for removing tasks, and otherwise just extend statestore?